.travis.yml

```diff
@@ -11,15 +11,15 @@ services:
 jdk:
   - oraclejdk11
 before_script:
-  - if [[ "$TRAVIS_BRANCH" = "master" && "$TRAVIS_EVENT_TYPE" = "push" ]]; then export
+  - if [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_EVENT_TYPE" == "push" ]]; then export
```

Comment:
Changing to double equals won't make any kind of difference, this is bash, not JS :P

Reply:
I guess you're right. I always thought that = and == work differently within [[ command, turns out it's the double quoting that makes things different. Although the essence of those changes is the 22nd line.

Reply:
There is a difference between them, but in this case it doesn't matter (i think it has to do with pattern matching).